### DISCUSSION TITLE
16x16 textures/fonts? also infinite worlds/crowded trees.

### DISCUSSION THREAD

**Discussion by DevelopmentPenguin** [MinicraftPlus/minicraft-plus-revived#654] [Q&A] [answered]
### 16x16 textures

Could you guys add 16x16 textures/fonts. I know it makes less of a
"mini" feel but i think it would be cooler that way. I'm not really wanting
this anyway. I just wanted to suggest it because i think it would be cool.
### infinite worlds

Also, could you add infinite worlds? It's better than having limited space.
Because it is really annoying on a 2d game when eventually you find an end.
I'm an og minecraft player. ive been playing for 12 and a half years now and i
found it really annoying when the world just stopped.

### crowded trees

could you add less crowded trees? its really annoying when
youre trapped in a circle of seemingly infinite trees. i know  it only happens
one in a while but its still reallly annoying.    what i mean: 

![Capture](https://github.com/MinicraftPlus/minicraft-plus-revived/assets/113947915/51fa0b05-5e60-40fd-8933-f95090b51a9c)

**Comment by Litorom** [comment]
The option for 16x16 textures may be available for resource packs in the future but that’s yet to be discussed. Infinite worlds is on the list for 2.4.0 when we stage out the rework of world saves. Trees are meant to be shown as a forest detail.

**Comment by BenCheung0422** [comment, marked as answer]
For the textures improvement about the sizes, it has already been mentioned in #394. For infinite world implementation, it has already been also mentioned in #317. About the tree textures, as mentioned above, it is used for forest appearance.

**Comment by DevelopmentPenguin** [comment]
okay thanks
